wedding/urls.py

```diff
@@ -18,8 +18,8 @@
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
-    url(r'', include('homeapp.urls')),
-    url(r'^search/', include('searchapp.urls')),
+    url(r'', include('homeapp.urls', namespace="home")),
```

Comment:
You must declare namespaces before using in django templates like 
{% url 'namespace: url' %}
i.e. {% url 'home:index' %} in your base.html